src/components/user/Asset.js

```diff
@@ -527,3 +551,13 @@ const Line = styled.div`
   height: 1px;
   background-color: ${LIGHT_GREY};
 `;
+
+const Message = styled.h4`
+  text-align: center;
+`;
+
+const CoinLink = styled(NavLink)`
+  text-decoration: none;
+  color: ${BLACK};
+  width: 20%;
```

Comment:
```suggestion
  width: 20%;
  text-decoration: none;
  color: ${BLACK};
```
레이아웃에 영향을 가는 순으로 적는다면 width는 이 위치에 와야할 거 같습니다 !